src/core/src/pass/serialize.cpp

```diff
@@ -709,6 +709,16 @@ const std::vector<Edge> create_edge_mapping(const std::unordered_map<ov::Node*,
 
 std::string get_opset_name(const ov::Node* n) {
     OPENVINO_ASSERT(n != nullptr);
+
+    // TODO: remove it one day: try to find opset name from RT info
```

Comment:
Minor: should we create a ticket for this TODO and leave its number in the comments?